read_structure_step/formats/mop/obabel.py

```diff
@@ -47,6 +52,82 @@
 }
 
 
+class OutputGrabber(object):
```

Comment:
Just happened to see this PR. I think you could use `contextlib.redirec_stdout` for this instead of writing your own class. https://docs.python.org/3/library/contextlib.html#contextlib.redirect_stdout

Reply:
Neat! Didn't know about that ... and didn't find it searching. I'll try it out ... and redo the release if it works, which I am sure it will.

Reply:
I learned about it from teaching my class at Berkeley :)